examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>1) "One side-effect of this pulse <strong>is</strong> <s>that</s> that the mean qubit frequencies will be shifted during the process."</p><p> </p><p>2) These phase shifts <strong>were</strong> measure... remove (). </p><p> </p><p>3) Additionally, we apply ...</p><p> </p><p>4) et al. </p><p> </p><p>5) Lets have an inline link to the title of the paper rather than just posting a link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>